utilities/transactions/transaction_base.cc

```diff
@@ -769,6 +801,18 @@ Status TransactionBaseImpl::RebuildFromWriteBatch(WriteBatch* src_batch) {
       return txn_->Put(db_->GetColumnFamilyHandle(cf), key, val);
     }
 
+    Status PutEntityCF(uint32_t cf, const Slice& key,
+                       const Slice& entity) override {
+      Slice entity_copy = entity;
+      WideColumns columns;
+      if (!WideColumnSerialization::Deserialize(entity_copy, columns).ok()) {
+        return Status::Corruption("Unable to deserialize entity",
```

Comment:
I'm wondering if it would be easier for the user to debug if we return the Status from `WideColumnSerialization::Deserialize()` as is instead of overwriting it with a generic failure message.

Reply:
Good point, will update